spaces/domains.json

```
@@ -617,7 +617,8 @@
   "vote.eggsforukraine.com": "eggsforukraine.eth",
   "vote.komorebinosekai.com": "komorebi-no-sekai.eth",
   "dao.getagump.org": "getagump.eth",
+  "vote.athame.finance":  "athamefinance.eth",
```

Please could you remove the extra space between key and value?
```suggestion
  "vote.athame.finance": "athamefinance.eth",
```